src/webots/vrml/WbProtoManager.cpp

```diff
@@ -254,6 +259,12 @@ QMap<QString, QString> WbProtoManager::undeclaredProtoNodes(const QString &filen
     return protoNodeList;
   // fill queue with nodes referenced by the world file
   queue << parser.protoNodeList();
+  const QString backwardsCompatibilityMessage =
+    tr("Please adapt your project to R2022b following these instructions: "
+       "https://github.com/cyberbotics/webots/wiki/How-to-adapt-your-world-or-PROTO-to-Webots-R2022b");
+
+  displayMissingDeclarations(backwardsCompatibilityMessage);
```

Comment:
Since here there's just one message, I think we can pass it directly without creating an intermediary variable